doc/tutorial/index.rst

```diff
@@ -854,17 +862,39 @@ new ``tftp.log``:
     #fields	ts	uid	id.orig_h	id.orig_p	id.resp_h	id.resp_p	wrq	fname	mode	uid_data	size	block_sent	block_acked	error_code	error_msg
     1367411051.972852	CKWH8L3AIekSHYzBU	192.168.0.253	50618	192.168.0.10	69	F	rfc1350.txt	octet	ClAr3P158Ei77Fql8h	24599	49	49	-	-
 
-The TFTP script also labels the service for the 2nd session session as
-"TFTP data", so we are now seeing this in ``conn.log``:
+The TFTP script also labels the 2nd session session as TFTP data by
```

Comment:
*session* is doubled. Since we are at it we could also write out *second*.
```suggestion
The TFTP script also labels the second session as TFTP data by
```